### DISCUSSION TITLE
Error reading files installed with  npm run refine swizzle

### DISCUSSION THREAD

**Discussion by LiaCiobanu001** [refinedev/refine#4441] [Q&A] [answered]
Hello, when I installed refine and I didn't have the sidebar, title, layout files, and with the command npm run refine swizzle I installed these files. But what changes I would not make on these files, it does not read them. I added the imports to App.tsx and my design overlaps. A little help please?

**Comment by salihozdemir** [comment, marked as answer]
Hey @LiaCiobanu001,

I'm sorry to hear that you're having trouble with the swizzle command. If you upgrade to the latest version of `@refinedev/mui` it should work as expected.

**Reply by LiaCiobanu001** [reply]
Thank you very much for the help.
